### DISCUSSION TITLE
Error Unzipped size must be smaller than 209927080 bytes.

### DISCUSSION THREAD

**Discussion by dnl4** [brefphp/laravel-bridge#110] [General]
I'm deploying a laravel application and got the error bellow:

```
Error:
CREATE_FAILED: WebLambdaFunction (AWS::Lambda::Function)
Resource handler returned message: "Unzipped size must be smaller than 209927080 bytes (Service: Lambda, Status Code: 400, Request ID: 52d36ff0-6343-4b88-80c4-3bf23c7b768e)" (RequestToken: 4b0d01cf-7d50-366b-4d83-ae3fb8e662ec, HandlerErrorCode: InvalidRequest)
```

Is possible to overcome it? I tried to add `runtime: docker` to `serverless.yml`, but didn't work.

**Comment by deleugpn** [comment]
You might want to take a look at https://bref.sh/docs/web-apps/docker.html and https://www.serverless.com/blog/container-support-for-lambda
